src/test/java/spoon/test/model/SwitchCaseTest.java

```diff
@@ -365,4 +367,52 @@ public void test_addCaseAt_throwsIndexOutOfBoundsException_whenPositionIsOutOfBo
 			assertThrows(IndexOutOfBoundsException.class, () -> switchExpression.addCaseAt(3, onlyCase));
 		}
 	}
+	
+	@DisplayName("should return the first case expression")
+	@Test
+	public void test_getCaseExpression_shouldReturnTheFirstCaseExpression() {
```

Comment:
Since you put the other tests in a nested type, I think this one should also be in a nested type, for consistency within this test class. It looks like all tests are categorized in nested types.

As you are testing different aspects of the same concept, I'd say put it into the new nested type you created. The other nested types don't categorize by method, but by concept, so that's fine in terms of consistency.

Reply:
> The other nested types don't categorize by method, but by concept, so that's fine in terms of consistency.

Right. Makes more sense.